ui/app/templates/partials/secret-list/item.hbs

```diff
@@ -8,12 +10,14 @@
   class="list-item-row"
   data-test-secret-link=item.id
   encode=true
+  queryParams=(secret-query-params backendModel.type)
 }}
   <div class="columns is-mobile">
     <div class="column is-10">
       <SecretLink
         @mode={{if item.isFolder "list" "show" }}
         @secret={{item.id}}
+        @queryParams={{query-params tab='actions'}}
```

Comment:
Does this one need to be conditional too?

Reply:
it sure did! i fixed that here. nice catch! https://github.com/hashicorp/vault/pull/8304/commits/06555d00b8c55bf3f48be7a33b21dc2fe56552f4